internal/linker/link_node_modules.ts

```diff
@@ -260,30 +260,45 @@ export async function main(args: string[], runfiles: Runfiles) {
   const links = [];
 
   const linkModule =
-      async (name: string, modulePath: string) => {
-    let target = runfiles.resolve(modulePath);
-
-    // It sucks that we have to do a FS call here.
-    // TODO: could we know which packages are statically linked??
-    if (!target || !await exists(target)) {
-      // Try the bin dir
+      async (name: string, kind: 'bin'|'src'|'runfiles', modulePath: string) => {
```

Comment:
nit: a comment with a brief explanation of the 3 choices: `bin`,`src` and `runfiles` for when we come back to this file 6 months from now?

Reply:
extracted a typedef and commented there

Reply:
🐙